apps/dashboard/src/components/topics/hooks/use-topics-url-state.ts

```diff
@@ -80,6 +80,8 @@ export const useTopicsUrlState = ({ after, before }: { after?: string; before?:
           } else {
             prev.delete('after');
           }
+        } else if (Object.prototype.hasOwnProperty.call(filter, 'after') && !filter.after) {
```

Comment:
is this the same as?
```suggestion
        } else if ('after' in filter && !filter.after) {
```

Reply:
This is more specific to the current object, without inherited properties. I might be able to use it here tho.